zsh-brew.plugin.zsh

```diff
@@ -15,21 +15,29 @@ if (( ! $+commands[brew] )); then
     done
 
     eval "$("$BREW_PATH" shellenv)"
+    # Tidy up environment
+    unset DEFAULT_BREW_PATHS
```

Comment:
The unset's are not required as the variables have been set to local, so are scoped to this file only.

Reply:
Thanks for the catch, not sure how I missed that. Removed the `unset` commands